_Classes/DeusEx/Classes/DeusExPlayer.uc

```diff
@@ -10361,7 +10360,7 @@ function UpdateCrosshairStyle()
         if (inHand.isA('DeusExWeapon') || dynamicCrosshair == 0)
     		root.hud.cross.SetBackground(Texture'CrossSquare');
         else if (inHand == None)
-    		root.hud.cross.SetBackground(Texture'CrossDot');
+    		root.hud.cross.SetBackground(Texture'RSDCrap.UserInterface.CrossDot');
```

Comment:
_Classes/DeusEx/Textures/CrossDot.bmp needs to be moved to _Classes/RSDCrap/Textures/CrossDot.bmp

Reply:
I believe I did move it

Reply:
Yeah I did, just need to delete it from _Classes/DeusEx/Textures/CrossDot.bmp